crawler/config/defaults.py

```diff
@@ -1,12 +1,17 @@
 # flake8: noqa
 import os
 
-from crawler.config.centres import *
+from crawler.config.centres import CENTRES as centres_config
 from crawler.config.logging import *
 from crawler.constants import SCHEDULER_JOB_ID_RUN_CRAWLER
 
 # setting here will overwrite those in 'centres.py'
 
+###
+# centres config objects
+###
+CENTRES = centres_config
```

Comment:
This is a bit weird, you're importing `CENTRES`, renaming to `centres_config`, then back to `CENTRES`? I'm sure there's a reason :)

Reply:
OK, if you have to import it here I think I would prefer an 'ignore' message for the linter, rather than this renaming - if that's a possibility?

Sorry I don't quite understand the first bit - what problem is importing CENTRES into this file solving? And why is this solution horrible? Happy to chat in person tomorrow if that's easier.

Reply:
I don't understand the first bit either.  See this comment: https://github.com/sanger/crawler/pull/523#issue-1165551521

All I can say is that the centre config wasn't being imported into defaults.py and I wonder whether it was doing so via some tenuous link that I broke when I fixed the way data is loaded in.  Probably only worked before by some amount of fortune, certainly not a direct import like I've included here.